src/Log/Migrations/CreateNewLogTable.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Give\Log\Migrations;
+
+use Give\Framework\Database\DB;
+use Give\Framework\Migrations\Contracts\Migration;
+use Give\Framework\Database\Exceptions\DatabaseQueryException;
+use Give\Framework\Migrations\Exceptions\DatabaseMigrationException;
+
+
+
+/**
+ * Class CreateNewLogTables
+ * @package Give\Log\Migrations
+ *
+ * @since 2.9.7
+ */
+class CreateNewLogTable extends Migration {
+
+	/**
+	 * @return string
+	 */
+	public static function id() {
+		return 'create_new_log_table';
+	}
+
+	/**
+	 * @return int
+	 */
+	public static function timestamp() {
+		return strtotime( '2021-01-28 12:00' );
+	}
+
+
+	public function run() {
+		global $wpdb;
```

Comment:
This can be removed, same with the other migrations.

Reply:
Resolved: a1e0cdd2